src/components/Select/index.ts

```diff
@@ -1,4 +1,5 @@
 import Select, { SelectProps } from './Select';
+import { SELECT_OVERLAY_STYLES } from './selectStyles';
 
-export { SelectProps };
+export { SelectProps, SELECT_OVERLAY_STYLES };
```

Comment:
nit: I think you should go ahead and import/export the type `SelectOverlayStyles` as well/in addition to this const.